### DISCUSSION TITLE
[Bug] Price configs don't work

### DISCUSSION THREAD

**Discussion by GhostsModels** [EtherealEmil/UsualScrap#38] [Ideas]
nuff said

**Comment by EtherealEmil** [comment]
All of them or just one? Either way I'll take a look.

**Reply by GhostsModels** [reply]
nevermind, I didn't realize but somehow you renamed the config file so I was editing the old one not realizing that it wouldn't actually change the files

**Comment by GhostsModels** [comment]
a change in config files had me editing the old one and not the new one
